### DISCUSSION TITLE
New chrome OS Update so it won't work

### DISCUSSION THREAD

**Discussion by Donovan5676** [3kh0/ext-remover#703] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
how can i fix it

**Comment by bartholemuew** [comment]
What won't work? You can't just say "It doesn't work plz fixxxxx" and not say what's broken.

**Reply by AshtonDavies** [reply]
It's probably a clueless youngling wanting to get the original extension remover working as if it wasn't patched almost a year ago in 106 starting the students vs. Google war we have today.
